### DISCUSSION TITLE
Sort ServApps Function / Minimal View

### DISCUSSION THREAD

**Discussion by j0nsgh** [azukaar/Cosmos-Server#335] [Ideas]
Hey there! Not sure if its a simple request but in the ServApps section, it would be nice to have a filter or sort function by name, or port, or network, etc.  Even a simple sort by name would be nice.  Right now I think it sorts by creation (newest shows first).

The other item is to have a minimal view on the ServApps page, basically name, icon, action buttons, and then a 'click for more' to drop the details, or 'show all', etc.

Anyway, just some thoughts.  Loving it so far and moving my items from CASAOS to this now actually.

**Comment by azukaar** [comment]
Thanks, appreciate it. 
Will close this discussion as the feature is not "up for debate" (as in, absolutely yes, filtering/sorting needs to be implemented) and we can keep track of it using #233 
Thanks!
